src/components/messenger/list/user-header/index.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { SettingsMenu } from '../../../settings-menu';
+
+import { bemClassName } from '../../../../lib/bem';
+import './styles.scss';
+
+const cn = bemClassName('user-header');
+
+export interface Properties {
+  userName: string;
+  userHandle: string;
+  userAvatarUrl: string;
+  userIsOnline: boolean;
+
+  includeUserSettings?: boolean;
+  onLogout?: () => void;
+}
+
+export class UserHeader extends React.Component<Properties> {
+  get userStatus(): 'active' | 'offline' {
+    return this.props.userIsOnline ? 'active' : 'offline';
+  }
+
+  render() {
+    return (
+      <div {...cn('')}>
+        {this.props.includeUserSettings && (
```

Comment:
while we're doing this, maybe remove this flag now? It is coming from `isMessengerFullScreen` global state, which is always true now

Reply:
Yes good shout! I'll remove that in a follow up PR